src/extension/metadata/src/main/java/org/geoserver/metadata/data/dto/impl/CommaSeperatedDeserializer.java

```diff
@@ -0,0 +1,22 @@
+/* (c) 2023 Open Source Geospatial Foundation - all rights reserved
+ * This code is licensed under the GPL 2.0 license, available at the root
+ * application directory.
+ */
+package org.geoserver.metadata.data.dto.impl;
+
+import com.fasterxml.jackson.core.JacksonException;
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.JsonDeserializer;
+import com.google.common.collect.Lists;
+import java.io.IOException;
+import java.util.List;
+
+public class CommaSeperatedDeserializer extends JsonDeserializer<List<String>> {
+
+    @Override
+    public List<String> deserialize(JsonParser p, DeserializationContext ctxt)
+            throws IOException, JacksonException {
+        return Lists.newArrayList(p.getText().split(","));
```

Comment:
You may want to `trim()` the split result to remove whitespace

Reply:
or split over ``"\\s*,\\s*"``

Reply:
adjusted